test/create.test.ts

```diff
@@ -27,6 +27,24 @@ describe("commands/create", function () {
     );
   });
 
+  test("throws if chain not provided", async function () {
```

Comment:
We had a test for "no chain". I added one, so now we have "no chain" and "bad chain".

Reply:
Great!